### ISSUE TITLE
Incorrect package name when creating a class if the package name contains "java" in it

### ISSUE THREAD

**Issue by Abiddarris** [AndroidIDEOfficial/AndroidIDE#1802]
### Issue Checklist

- [X] I confirm that this issue is not a duplicate and hasn't been reported before.
- [X] I agree to follow the project's code of conduct.
- [X] I have checked and verified that I am using the latest version of AndroidIDE from GitHub or F-Droid.

### Additional terms

- [X] I understand that providing incomplete or inaccurate information may result in the issue being closed.
- [X] I agree to provide all requested details promptly for a comprehensive bug report.

### Issue explanation

Steps to reproduce:

1.Create a package "com.javarunner.name".
2. Create a class named "Test".

Result:
```
package runner.name;

public class Test {
}
```

### Expected behavior

```
package com.javarunner.name;

public class Test {
}
```

### Version name

Version : v2.7.1-beta (27102)
CI Build : true
Branch : main
Commit : 55bb46b31
Variant : arm64-v8a (release)
Build type : OFFICIAL
F-Droid Build : false
F-Droid Version : null
F-Droid Version code : -1
SDK Version : 31
Supported ABIs : [arm64-v8a, armeabi-v7a, armeabi]
Manufacturer : samsung
Device : SM-M127F

### Download source

GitHub

**Comment by ABirHAsan123-beep** [comment]
#1802

**Comment by itsaky** [comment]
Fixed with #1863
